Tasks/NuGetInstaller/vsonugethelper.ts

```diff
@@ -0,0 +1,184 @@
+﻿/// <reference path="../../definitions/node.d.ts"/>
```

Comment:
The vso-agent-tasks repo has an internal module concept that you might find helpful. It solves the "internal shared code" problem for tasks authored within this repo. As an example, you can take a look at how the msbuild/vsbuild task share common code: 1) entry in /common.json 2) shared code in /Common/MSBuildHelpers/ 3) build process copies the shared code into a ps_modules folder under /_build/Tasks/MSBuild and /_build/Tasks/VSBuild.